src/smc-util/db-schema.coffee

```diff
@@ -385,6 +385,42 @@ schema.client_error_log =
         event : []
     pg_indexes : ['time', 'event']
 
+schema.webapp_errors =
+    primary_key : 'id'
+    durability : 'soft' # loss of some log data not serious, since used only for analytics
+    fields:
+        id           : type : 'uuid'
+        account_id   : type : 'uuid'
+        name         : type : 'string'
+        message      : type : 'string'
+        comment      : type : 'string'
+        stacktrace   : type : 'string'
+        file         : type : 'string'
+        lineNumber   : type : 'integer'
+        columnNumber : type : 'integer'
+        severity     : type : 'string'
+        browser      : type : 'string'
+        mobile       : type : 'boolean'
+        responsive   : type : 'boolean'
+        user_agent   : type : 'string'
+        path         : type : 'text'
+        smc_version  : type : 'string'
+        build_date   : type : 'string'
+        smc_git_rev  : type : 'string'
+        uptime       : type : 'string'
+        start_time   : type : 'timestamp'
+        time         : type : 'timestamp'
+    indexes:
```

Comment:
you can delete the indexes thing -- it's 100% only used by the rethinkdb stuff, which I'm going to complete delete asap.